### ISSUE TITLE
Docker image does not have arm64 version, causing  exec format error

### ISSUE THREAD

**Issue by xRahul** [valeriansaliou/sonic#323]
Hi,

Possible to build & release arm64 version also? Currently unable to use on raspberry pi 5

**Comment by pirate** [comment]
ArchiveBox maintains an arm64 Docker build you can use: [`archivebox/sonic:latest`](https://hub.docker.com/r/archivebox/sonic) https://hub.docker.com/r/archivebox/sonic

**Comment by JulianWgs** [comment]
Native ARM64 runners are available on GitHub now:
https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/

The ARM64 image build was removed from this repository, because of poor performance of QEMU: https://github.com/valeriansaliou/sonic/blob/0a6e04efb4570ee24d0d1368d1e6c986555a367a/server/CHANGELOG.md#148-2023-12-14

Now that native runners are available, it is probably a good idea to add them again. Unfortunately I am not so familiar with GitHub action to do it.

**Comment by RemiBardon** [comment]
I recently made improvements to the CI (#348, #351), I will add support for more architectures in a future release.

**Comment by hiSandog** [comment]
For the Docker arm64 request, the key maintenance question is whether release builds can publish a manifest list for both `linux/amd64` and `linux/arm64`. If the binary already builds cleanly on arm64, adding buildx-based multi-arch publishing would solve the `exec format error` class directly. If not, the image tag should probably document architecture support to avoid surprises on ARM hosts.

**Comment by RemiBardon** [comment]
Yep, that shouldn't be an issue. The docker/build-push-action already pushes a manifest list (OCI image index). Sonic builds cleanly on arm64, that's what we use for dev anyway.

I was about to make a release of the server (just have to release our new official Rust client first), I'll squeeze in the addition of support for arm64 🙂

**Comment by RemiBardon** [comment]
Closed by 628a7d1fcc579c5dd1d30c3c0782549a61de0320, available since v1.7.4
